src/style.css

```
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: 'Roboto', sans-serif;
+  src: url();
+}
+
+html {
+  font-family: 'Roboto', sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: #E5E5E5;
+}
+
+.menu-bar {
+  display: flex;
+  min-width: 1200px;
+  height: 60px;
+  background-color: white;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.05);
+  margin: 20px
+}
+
+.menu-logo {
+  display: flex;
+  flex-basis: 250px;
+  flex-shrink: 1;
+  flex-grow: 1;
+  padding: 10px 50px;
+}
+
+.nav-bar {
+  display: flex;
+  flex-grow: 1;
+  justify-content: flex-end;
+}
+
+.menu-bar--small {
+  display: flex;
+  width: 1024px;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  align-items: center;
+  font-size: 12px;
+  padding: 0;
+}
+
+.nav-list-item {
+  display: flex;
+  flex-grow: 1;
+  text-transform: uppercase;
+  margin: 10px;
+}
+
+.nav-list-item:last-child {
+  margin-right: 50px;
+}
+
+.nav-list-link {
+  color:black;
+  text-decoration: none;
+  padding: 23px 0;
+}
+
+.nav-list-link:active {
+  color: #00ACDC;
+}
+
+.nav-list-link:hover {
+  color: #00ACDC;
+}
+
+.is-active {
+  color: #00ACDC;
+  position: relative;
+}
+
+.is-active::after {
+  position: absolute;
+  content: '';
+  height: 4px;
+  width: 37px;
```

```suggestion
  width: 100%;
```
так лінія буде правильної ширини для усіх слів